dockerfiles/sv_germline_granite/scripts/test/test_SV_confidence.py

```diff
@@ -0,0 +1,82 @@
+#################################################################
+#   Libraries
+#################################################################
+import sys, os
+import pytest
+import shutil
+from subprocess import Popen
+from granite.lib import vcf_parser
+
+
+confidence = __import__("SV_confidence")
+
+
+def test_manta(tmp_path):
+    # Variables and Run
+    args = {
+        "input": "test/files/confidence_classes_manta_in.vcf.gz",
+        "output": f"{tmp_path}/output.vcf",
+        "tool": "manta",
+    }
+
+    confidence.main(args)
+    a = os.popen(f"bgzip -c -d {tmp_path}/output.vcf.gz")
+    b = os.popen("bgzip -c -d test/files/confidence_classes_manta_out.vcf.gz")
+
+    # Test
+    assert [row for row in a.read()] == [row for row in b.read()]
+
+
+def test_manta_multiple_gentype_ids(tmp_path):
+    # Variables and Run
+    args = {
+        "input": "test/files/confidence_classes_manta_3_genotype_ids_in.vcf.gz",
```

Comment:
Great that you thought about careful tests. Could you explain what is different in 'test_manta_multiple_gentype_ids' compared to 'test_manta' and the test files?

Reply:
In `test_manta` we test a VCF file that stores various combinations of possible variants classes and there is only one sample tested. In `test_manta_multiple_gentype_ids` we test a VCF file that stores several variants for three samples to check if we can evaluate confidence classes for all the samples provided in the file. The same approach is applied for the BIC-Seq2 tests. 

I will leave comments on this in the script to make it clearer.

Reply:
In` test_manta_multiple_gentype_ids `, does it mean that each of 3 samples will get its own confidence class for the same variant?